### ISSUE TITLE
Scrolling would be really great

### ISSUE THREAD

**Issue by Bluebie** [BlueM/cliclick#10]
It'd be heaps nice if there was a way to issue scrolling commands - ideally the smooth scrolling kind, not the chunky click mouse wheel variety. Is this a possibility?

**Comment by wfaulk** [comment]
Implemented in #22

**Comment by smileytechguy** [comment]
should this be closed as its merged?

**Comment by BlueM** [comment]
Yes, the PR was merged, but afterwards I noticed issues that I had not noticed while doing some testing before the merge. This was the reason why I reverted the merge and moved the code to a new branch [scrolling](https://github.com/BlueM/cliclick/tree/scrolling), where it has been untouched by anyone for the last roughly 1,5 years. So, for now, cliclick does not support scrolling events.

**Comment by Zlitus** [comment]
I would love this feature. This would allow, on the Macbook with Notches, to use the macOS zoom from the center of the screen, and put videos "really in full screen" (Today, Apple always add a black bar on top of the screen and you can't disabled it, even for video).

Here would be my scenario:

1) Put the mouse cursor at the center of the screen
2) Press CTRL
3) Use mousewheel forward for 0.3s

**Comment by qizidog** [comment]
Ping! Will the mouse scrolling feature be actually merged into the main branch in future?

**Comment by BlueM** [comment]
@qizidog: I haven’t even taken a look at the branch in the last six years, therefore I don’t think so. I don’t recall exactly which problem(s) I discovered (cf. my last comment in this issue), but as far as I remember, it was nothing that gave me the impression it was easy or straightforward to fix. So for the moment, I tend to say: sorry, no.

**Comment by dandepeched** [comment]
would love to see it as well

**Comment by gluebyte** [comment]
iPhone Mirroring (introduced in macOS Sequoia this year) seems to only recognize wheel scrolling as scrolling, e.g. I cannot scroll a Safari page using `dd` and `du`. It would be great for iPhone automation if scrolling is supported
